### ISSUE TITLE
WCAG and W3C validation problems

### ISSUE THREAD

**Issue by nikitai** [yannickoo/minimal_share#22]
If label is empty then we fail WCAG and W3C validations.

Solution:
alter the minimal_share function in the end, then we will be albe to insert empty DOM element for example.

**Comment by nikitai** [comment]
`diff --git a/modules/contrib/minimal_share/minimal_share.module b/modules/contrib/minimal_share/minimal_share.module
index c469409..78b4200 100644
--- a/modules/contrib/minimal_share/minimal_share.module
+++ b/modules/contrib/minimal_share/minimal_share.module
@@ -334,6 +334,7 @@ function minimal_share($url = '', $title = '', $settings = array()) {
       }
     }

+    drupal_alter('minimal_share', $service['label'], $service['url'], $link_options);
     $element['#value'] .= l($service['label'], $service['url'], $link_options);
   }`
